package-system/PhysX5/FindPhysX5.cmake

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(MY_NAME "PhysX5")
```

Comment:
```suggestion
set(PACKAGE_NAME "PhysX5")
```

Reply:
I'm reluctant to change that pattern now. I think in every 3P package's Find*.cmake it uses the `MY_NAME` pattern.